### PR TITLE
Remove grunt

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

Removes the vestiges of `grunt` from frontend

## What is the value of this and can you measure success?

- 1 fewer task runner
- we control our processes
- asset compilation is ~2.4x faster (from around `1m 02s` to around `26s` on my machine)

## Does this affect other platforms - Amp, Apps, etc?

hopefully no!

### CODE REVIEW

File: docs/01-start-here/01-installation-steps.md [hunk 1]
```diff
@@ -97,12 +97,6 @@ Mac:
 ```bash
 brew install node@6
 ```
-
-#### Grunt (build tool)
-
-Ubuntu/Mac:
-```
-npm -g install grunt-cli
 ```
```
Comment (SiAdcock) [inline docs/01-start-here/01-installation-steps.md]: Need to kill these backticks or the formatting of the markdown file gets messed up

Comment (sndrs) [inline docs/01-start-here/01-installation-steps.md]: doh!

**Comment by SiAdcock** [comment]
Doing a bit of grepping, there's still refs to Grunt in [tools/screenshots/README.md](https://github.com/guardian/frontend/blob/remove-grunt/tools/screenshots/README.md)

Otherwise, looks awesome!

**Comment by sndrs** [comment]
hmmm thought i'd caught em all! that'll go with #15217, so i might leave that one? will double check for other references though

**Comment by SiAdcock** [comment]
Nice one 👍 👍 👍

**Pull request merged**